Flagship/Enums/InstanceType.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Flagship.Enums
+{
+    public enum InstanceType
+    {
+        SINGLE_INSTANCE,
```

Comment:
I don't see anywhere where SINGLE_INSTANCE is used set to a visitor instance)

Reply:
In VisitorBuilder